server/models.py

```diff
@@ -776,13 +776,20 @@ class Backup(Model):
     id = db.Column(db.Integer, primary_key=True)
 
     submitter_id = db.Column(db.ForeignKey("user.id"), nullable=False)
+    # NULL if same as submitter
+    creator_id = db.Column(db.ForeignKey("user.id"), nullable=True)
     assignment_id = db.Column(db.ForeignKey("assignment.id"), nullable=False)
     submit = db.Column(db.Boolean(), nullable=False, default=False)
     flagged = db.Column(db.Boolean(), nullable=False, default=False)
 
     extension = db.Column(db.Boolean(), default=False)
 
-    submitter = db.relationship("User")
+    # The time we should treat this backup as being submitten. If NULL, use
+    # the `created` timestamp instead.
+    submission_time = db.Column(db.DateTime(timezone=True), nullable=True)
```

Comment:
Do you think it makes sense to have a hybrid_property called `time` and it'll choose which to report? 

I can't see where we would use it right now - but it might be useful if we just want to display the time we treated it as.